README.md

```diff
@@ -115,7 +115,12 @@ Apps should have their directory names prefixed with `app-`, e.g. `/project` bec
 
 ## Production deployment
 
-Deploys to production are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image is created which installs and builds the `lib-` packages, and that's used as a base image for creating `app-` images, which are then deployed to Kubernetes.
+Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image (`zooniverse/front-end-monorepo`) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages. 
```

Comment:
```suggestion
Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image ([`zooniverse/front-end-monorepo`](https://hub.docker.com/r/zooniverse/front-end-monorepo)) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages. 
```

Reply:
Adds a link on Docker Hub

Reply:
This text is pretty much all deleted in #1126.